packages/base/src/plugin.ts

```
@@ -2,6 +2,7 @@ import { SecretsVault } from './interface.ts';
 
 export class SecretsVaultPlugin extends SecretsVault {
 	async get(key: string) {
+		if (key.includes('/')) key = key.replaceAll('/', '_');
```

Biome

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)